vita3k/renderer/src/gl/texture.cpp

```diff
@@ -283,5 +285,67 @@ void upload_bound_texture(const SceGxmTexture &gxm_texture, const MemState &mem)
     }
 }
 
+// Dumps bound texture to a file
+void dump(const SceGxmTexture &gxm_texture, const MemState &mem, const std::string &parameter_name, const std::string &base_path, const std::string &title_id, Sha256Hash program_hash) {
+    static uint32_t gg_tex_index = 0;
```

Comment:
you meant `g_tex_index`?
`g_` is supposed to be for globals and below you use it for a local instead
not sure what "gg" is

Reply:
Absolutely right. I fixed it.